src/generator/index.tsx

```diff
@@ -66,25 +67,17 @@ export default async function render({ messages, channel, callbacks, ...options
         <title>{channel.isDMBased() ? 'Direct Messages' : channel.name}</title>
 
         {/* message reference handler */}
-        <script
-          dangerouslySetInnerHTML={{
-            __html: scrollToMessage,
-          }}
-        />
+        <script>{scrollToMessage}</script>
 
         {!options.hydrate && (
           <>
             {/* profiles */}
-            <script
-              dangerouslySetInnerHTML={{
-                __html: `window.$discordMessage={profiles:${JSON.stringify(await profiles)}}`,
-              }}
-            ></script>
+            <script>{`window.$discordMessage={profiles:${JSON.stringify(await profiles)}}`}</script>
```

Comment:
`profiles` is already awaited now, so no need for this redundant `await` inside JSON.stringify